index.html

```diff
@@ -8,7 +8,11 @@
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Playfair+Display:wght@500&family=Poppins&display=swap" rel="stylesheet">
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./headline.css">
+    <link rel="stylesheet" href="works.css">
+    <link rel="stylesheet" href="about.css">
+    <link rel="stylesheet" href="contact.css">
+    <link rel="stylesheet" href="footer.css">
 </head>
 <body>
```

Comment:
- [ ] The minimum mobile screen size should be `375`px, but when the screen size is `375px` your design have some issues(white space to the right of the page, some images are not aligned correctly,...), please kindly make sure your design stick as much as possible to the mobile Figma design.


> |     | Your design                      | Figma design                      |     |
> | --- | ----------------------------------- | ------------------------------------ | --- |
> |     | <img width="190" alt="Screen Shot 2022-10-29 at 15 38 03" src="https://user-images.githubusercontent.com/74501523/198834629-f555ad36-ced5-4c1a-acdb-a7e354fdec7f.png">  | <img width="160" alt="Screen Shot 2022-10-29 at 15 37 20" src="https://user-images.githubusercontent.com/74501523/198834654-bd2ef2f5-36e2-433d-b2de-2c7ae7f79243.png"> |